app/api/csv/specs/csvLoaderThesauri.spec.js

```diff
@@ -118,5 +118,136 @@ describe('csvLoader thesauri', () => {
       ]);
       mockedFile.mockRestore();
     });
+
+    describe('nesting', () => {
```

Comment:
Please consider adding one test case, which checks the value of translations.

Reply:
sure

Reply:
done